tests/test_custom_checkers/test_inconsistent_returns.py

```diff
@@ -153,3 +154,92 @@ def func():
             ignore_position=True,
         ):
             self.checker.visit_functiondef(func_node)
+
+
+class TestInconsistentReturnCheckerZ3Option(pylint.testutils.CheckerTestCase):
+    CHECKER_CLASS = InconsistentReturnChecker
+
+    def setup_method(self) -> None:
+        super().setup_method()
+        self.linter.config.z3 = True
```

Comment:
There's a more idiomatic way to pass in configuration options to `CheckerTestCase` subclasses; see other test suites for examples.